services/helpers.go

```diff
@@ -39,3 +39,27 @@ func PrepareQueries(c echo.Context) (rawQuery string, flag *string) {
 func GetDidParam(c echo.Context) (string, error) {
 	return url.QueryUnescape(c.Param("did"))
 }
+
+func ExtractMediaTypeParams(mediaType string) map[string]string {
```

Comment:
As far as I understand, this is just stripping out the `q` values without taking it into consideration for priority order?

Reply:
Added Priority handling in accept headers now